Workflow.Rmd

```diff
@@ -483,6 +751,34 @@ likely to be satisfied. Yet when our data are of a more limited size, or if the
 the sample size, then we will need to make some further assumptions in order to say anything meaningful about
 the random variables in our model. That's where Bayesian modeling comes in. 
 
+**How do we use random variables in our modeling?**
+
```

Comment:
We use the idea of a random variable for

- It is valuable to treat objects of interest as random variables as it allows us to understand their distributions in a tangible way. It's a convenient way of summarizing the object of interest. What are the objects of interest?

- outcomes which we want to model
- to summarize our uncertainty of model unknowns (which are fixed values)